package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "shopify_app",
-  "version": "14.2.0",
+  "version": "11.4.0",
```

Comment:
I think there may have a been an issue with the version number here. In any case, we usually bump the version when we're going through the actual release process, so you shouldn't need to it in this PR.

Reply:
Thanks. I'll revert this.